source/ECS/Storage.hpp

```diff
@@ -116,7 +119,7 @@ namespace ECS
                 if constexpr (!std::is_same_v<Entity, std::decay_t<ComponentType>>) // Ignore any Entity params supplied.
                     componentBitset.set(get_ID<ComponentType>());
             };
-            (setComponentBit.operator()<ComponentTypes>(), ...);
+            (setComponentBit.template operator()<ComponentTypes>(), ...);
```

Comment:
This is a strange one, I already didn't understand why the operator() part was necessary...